llama_stack/apis/agents/agents.py

```diff
@@ -294,8 +137,9 @@ class AgentConfigCommon(BaseModel):
 
     input_shields: Optional[List[str]] = Field(default_factory=list)
     output_shields: Optional[List[str]] = Field(default_factory=list)
-
-    tools: Optional[List[AgentToolDefinition]] = Field(default_factory=list)
+    available_tools: Optional[List[str]] = Field(default_factory=list)
```

Comment:
can we call them: registered_tools - even custom_tools are available tools?

Reply:
i think we can just name it tool_names to make it clear. WDYT?